cpp/oneapi/dal/graph/detail/undirected_adjacency_array_graph_impl.hpp

```diff
@@ -19,16 +19,19 @@
 #include "oneapi/dal/common.hpp"
 #include "oneapi/dal/detail/common.hpp"
 #include "oneapi/dal/graph/detail/graph_container.hpp"
+#include "oneapi/dal/graph/graph_common.hpp"
 
 namespace oneapi::dal::preview::detail {
 
 template <typename Graph>
-ONEDAL_EXPORT typename Graph::pimpl& get_impl(Graph& graph) {
+ONEDAL_EXPORT oneapi::dal::detail::pimpl<typename graph_traits<Graph>::impl_type>& get_impl(
```

Comment:
Why `dal::detail::get_impl` declared in `dal/detail/common.hpp` cannot be used instead?

Reply:
/intelci: run

Reply:
Updated.